torchvision/csrc/cuda/deform_conv2d_cuda.cu

```diff
@@ -832,7 +625,7 @@ static void compute_grad_offset_and_mask(
   }
 }
 
-static std::tuple<at::Tensor, at::Tensor, at::Tensor> deform_conv2d_backward_input_cuda(
+std::tuple<at::Tensor, at::Tensor, at::Tensor> backward_gradient_inputs(
```

Comment:
Removing unnecessary prefixing of technique names, since with the namespaces we won't have conflicts any more.

Reply:
Hum, I'm unsure about this change, and I think it would be preferable to keep the template of `{opname}_backward_{type}` for the names, as it [is done for PyTorch](https://github.com/pytorch/pytorch/blob/759a548d6eac8e38e12f6c144cee7b7c504591d4/aten/src/ATen/native/ConvolutionMM2d.cpp#L554)

Reply:
This is not the the template you think it is... That's why I rename this function here. 
Check again the file; you will see that there is already the `{opname}_backward_{type}` method that you want. This is an internal method which I don't believe should use this convention.